shared/getting-started/deployingCode/nodejs.md

```diff
@@ -1,5 +1,6 @@
-A nice first project to get started is a simple [Express.js][expressjs] web server which, will serve a static page on port `:80`. All the project source code can be found [here on GitHub][simple-server-node-link]. Download a zipped file of the project [here][github-download], and when downloaded, unzip the file and open a terminal in the root of the project directory.
+A nice first project to get started is a simple [Express.js][expressjs] web server which, will serve a static page on port `:80`. All the project source code can be found [here on GitHub][simple-server-node-link]. Download a zipped file of the project [here][github-download-js], and when downloaded, unzip the file and open a terminal in the root of the project directory. For a TypeScript version of the project please go [here][github-download-ts].
 
 [simple-server-node-link]:{{ $links.githubLabs }}/simple-server-node
```

Comment:
```suggestion
[balena-node-hello-world]:{{ $links.githubLabs }}/balena-node-hello-world
```
Because the `simple-server-node` repo was renamed `balena-node-hello-world`, try visiting the following to confirm: 
* https://github.com/balenalabs/simple-server-node
* https://github.com/balenalabs/balena-node-hello-world
* https://github.com/balena-io-examples/balena-node-hello-world

I guess that `{{ $links.githubLabs }}` points to `github.com/balenalabs`. The most correct would be to have something like `{{ $links.githubExamples }}` -- to be created if it does not exist -- that pointed to `github.com/balena-io-examples`:

```suggestion
[balena-node-hello-world]:{{ $links.githubExamples }}/balena-node-hello-world
```

Reply:
I am not sure where the redirect is happening, but yeah, `$links.githubLabs` redirects to `$links.githubExamples`. Happy to change those references as well, done.

Reply:
I believe that GitHub does the redirect, as a configurable repo setting.